app/api/routes.py

```
@@ -32,6 +33,35 @@ class RecordCreateResponse(BaseModel):
     id: int
 
 
+@router.get("/api/v1/records/weekly-average",
+            tags=["복막투석기록"],
+            summary="주간 기록 데이터 평균 조회",
+            description="특정 날짜가 포함된 주의 (월요일 ~ 일요일) 환자 기록 데이터의 평균을 계산하여 반환합니다.",
+            response_model=WeeklyAverageResponse,
+            )
+def get_weekly_average(
+        current_user: User = Depends(AuthTokenDep),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)